microsoft-edge/devtools-guide-chromium/whats-new/2023/03/devtools-111.md

```diff
@@ -0,0 +1,155 @@
+* [Securely debug original code by using Azure Artifacts symbol server source maps](../../../javascript/consume-source-maps-from-azure.md)
+* [Record performance](../../../evaluate-performance/reference.md#record-performance) in _Performance features reference_.
+
+
+<!-- ====================================================================== -->
+## Styles tab easier to use in Focus Mode
+
+<!-- Subtitle: In previous versions of Microsoft Edge, the Styles pane of the Elements tool didn't provide easy access to all of the sections within it. This issue has now been fixed in Microsoft Edge 111. -->
+
+In previous versions of Microsoft Edge, in **Focus Mode**, the **Elements** tool didn't provide an easy way to access all of the tabs that are grouped with the **Styles** tab.  In Microsoft Edge 111, this issue has been fixed.  When the **Styles** pane has a narrow viewport and can't display all of the other available tabs, click the **More tabs** expander icon (![More tabs expander icon](./devtools-111-images/more-tools-expand-icon.png)).
+
+![The collapsed tab-set in the Styles pane in Focus Mode](./devtools-111-images/collapsed-menu-for-styles-pane.png)
+
+To display only a few of the tabs that are grouped with the **Styles** tab, click the **More tabs** collapser icon (![More tabs collapser icon](./devtools-111-images/more-tools-collapse-icon.png)).
+
+![The expanded tab-set in the Styles pane in Focus Mode](./devtools-111-images/expanded-menu-for-styles-pane.png)
+
+See also:
+* [CSS features reference](../../../css/reference.md)
+* [Simplify DevTools using Focus Mode](../../../experimental-features/focus-mode.md)
+
+
+<!-- ====================================================================== -->
+## Copy Selector Stats table from Performance tool to Excel
+
+<!-- Subtitle: In Microsoft Edge 111, you can copy and paste Selector Stats data from the Performance tool for aggregate selector performance analysis. -->
+
+Microsoft Edge 109 introduced the **Selector Stats** feature in the **Performance** tool.  With the **Enable advanced rendering instrumentation (slow)** setting turned on in the **Performance** tool, selecting any **Recalculate Styles** event in the flame chart adds a **Selector Stats** tab in the bottom pane.  See [Debug long-running Recalculate Style events with new selector stats](../../2023/01/devtools-109.md#debug-long-running-recalculate-style-events-with-new-selector-stats) in _What's new in DevTools (Microsoft Edge 109)_.
+
+In Microsoft Edge 111, you can now export the **Selector Stats** table for each **Recalculate Styles** event to a Microsoft Excel workbook.  You can then use Excel to do aggregate selector performance analysis, and identify the most costly selectors over the lifetime of the profile.
+
+To export the **Selector Stats** table:
+
+1. Click the upper left cell with the mouse, and then drag to select to the bottom right cell.  Or, click once or twice in a cell, and then press **Ctrl+A**.
```

Comment:
I know you found a bug here and I'm not eager to show that to our readers. How about:
```suggestion
1. Click the upper left cell with the mouse, and then drag to select to the bottom right cell. 
```